File: JS/8498_cm.js
function game(field) {
	let ans = 0;
	let field_s = [];
	let map_s = [];
	let map = [
		'0 1 0 1 0 1 0 1',
		'1 0 1 0 1 0 1 0',
		'0 1 0 1 0 1 0 1',
		'1 0 1 0 1 0 1 0',
		'0 1 0 1 0 1 0 1',
		'1 0 1 0 1 0 1 0',
		'0 1 0 1 0 1 0 1',
		'1 0 1 0 1 0 1 0'
	];

	for (let i = 0; i < 8; i++) {
		let row = map[i].split(' ').filter(item => item !== ' ');
		map_s.push(row);
	}

	for (let i = 0; i < 8; i++) {
		let row = [...map[i]];
		field_s.push(row);
	}

	for (let i = 0; i < 8; i++) {
		for (let j = 0; j < 8; j++) {
			if (field_s[i][j] == 'F' && map_s[i][j] == '0') {
				ans += 1;
			}
		}
	}

	console.log(ans);
	return ans;
}

field = [
	'TFFTTFTT', 'FTFTFTTF', 'TTTFTFTT', 'TTTTFFTT',
	'TFTTFTFT', 'TTTFFTTF', 'TFTFTFTT', 'TFTFTTFT'
];

let tmp = '0 t';

console.log(tmp.split(' '));

game(field);

File: Py/8498_cm.py
# 입력이 적어서 하나씩 세도 됨
# 맵이 0이고 field가 t이면 += 1

import sys

# field = [map(str, sys.stdin.readline().split()) for _ in range(8)]
# field = [item.split() for item in field]

field = []

for _ in range(8):
	row = list(sys.stdin.readline().strip())
	field.append(row)


map = [
	'0 1 0 1 0 1 0 1',
	'1 0 1 0 1 0 1 0',
	'0 1 0 1 0 1 0 1',
	'1 0 1 0 1 0 1 0',
	'0 1 0 1 0 1 0 1',
	'1 0 1 0 1 0 1 0',
	'0 1 0 1 0 1 0 1',
	'1 0 1 0 1 0 1 0'
]
map = [item.split(' ') for item in map]
ans = 0

for i in range(8):
	for j in range(8):
		if (field[i][j] == 'F' and map[i][j] == '0'):
			ans += 1

print(ans)